src/sageworks/artifacts/endpoints/endpoint.py

```diff
@@ -452,7 +469,7 @@ def shap_values(model, X: pd.DataFrame) -> pd.DataFrame:
         # Note: For Tree-based models like decision trees, random forests, XGBoost, LightGBM,
         explainer = shap.TreeExplainer(model)
         shap_vals = explainer.shap_values(X)
-        return pd.DataFrame(shap_vals, columns=X.columns)
+        return shap_vals
```

Comment:
Update the return value to include the Union[] like you did in other places

Reply:
So this now actually returns an np.array - I'll make the change.

The np.array here ends up being (# of prediction classes x # samples x # features), which later gets split into a separate dataframe for each prediction class, so that's when the Union of the list[pd.DataFrame] type comes in.